tests/test_utilblocks.py

```diff
@@ -0,0 +1,70 @@
+# Standard
+from unittest.mock import MagicMock, patch
+import unittest
+
+# Third Party
+from datasets import Dataset, Features, Value
+
+# First Party
+from src.instructlab.sdg.utilblocks import (
+    DuplicateColumns,
+    FlattenColumnsBlock,
+    RenameColumns,
+    SetToMajorityValue,
+)
+
+
+class TestUtilBlock(unittest.TestCase):
+    def setUp(self):
+        self.mock_ctx = MagicMock()
+        self.mock_pipe = MagicMock()
+
+        self.dataset = Dataset.from_dict(
+            {
+                "name": ["Tomato", "Rose", "Carrot", "Basil"],
+                "type": ["Fruit", "Flower", "Vegetable", "Herb"],
+                "edible": ["yes", "no", "yes", "yes"],
+            }
+        )
+
+    def test_dup(self):
+        dup_block = DuplicateColumns(
+            self.mock_ctx, self.mock_pipe, "blockname", columns_map={"edible": "caneat"}
+        )
+        assert "caneat" not in self.dataset.column_names
+        new_samples = dup_block.generate(self.dataset)
+        assert "edible" in new_samples.column_names
+        assert "caneat" in new_samples.column_names
```

Comment:
Another test here could be to validate that the two columns have equivalent values, as well.